docs/utils/data.rst

```diff
@@ -0,0 +1,220 @@
+.. _utils-data:
+
+***************************************************
+Downloadable data management (`astropy.utils.data`)
+***************************************************
+
+Introduction
+============
+
+A number of Astropy's tools work with data sets that are either awkwardly
+large (for example `~astropy.coordinates.solar_system_ephemeris`) or
+regularly updated (for example `~astropy.utils.iers.IERS_B`) or both
+(for example `~astropy.utils.iers.IERS_A`). What's more, this kind of
+data - authoritative data made available on the Web, and possibly updated
+from time to time - is reasonably common in astronomy. Astropy therefore
+provides some tools for working with such data.
+
+The primary tool for this is the Astropy *cache*. This is a repository
+of downloaded data, indexed by the URL where it was obtained. The tool
+`~astropy.utils.data.download_file` and various other things built upon
+it can use this cache to request the contents of a URL, and (if they
+choose to use the cache) the data will only be downloaded if it isn't
+already present in the cache. Of course they can be instructed to obtain
+a new copy of data that has been updated.
+
+The astropy cache is stored in a centralized place (on
+UNIX machines it is ``$HOME/.astropy/cache``, or if the
+environment variable ``XDG_CACHE_HOME`` is set, the cache is in
+``$XDG_CACHE_HOME/astropy``). This centralization means that the cache
+is persistent and shared between all Astropy runs in any virtualenv
+by your user on one machine (possibly more if your home directory is
+shared between multiple machines). This can dramatically accelerate
+Astropy operations and reduce load on servers, like those of the IERS,
+that were not designed for heavy Web traffic. If you find the cache has
+corrupted or out-of-date data in it, you can remove an entry or clear
+the whole thing with `~astropy.utils.data.clear_download_cache`.
+
+The files in the download cache directory are named according to a
+cryptographic hash of their contents (currently MD5, so malevolent entities can
```

Comment:
Should we really point it out so clearly and give these "malevolent entities" ideas in the user-facing doc? If there is nothing the user can do to avoid this, maybe not worth mentioning here?

Reply:
When I wrote this it looked like maybe my PR to switch to a not-yet-broken hash would get in.